### ISSUE TITLE
wwsympa: empty error message when trying to access list info page with insufficient rights

### ISSUE THREAD

**Issue by drkspc** [sympa-community/sympa#1487]
Accessing the info page of a concealed list without sufficient rights shows an empty error message.

Version
-------
Sympa 6.2.68

Installation method
-------------------
FreeBSD ports

Expected behavior
-----------------
The error message informs the user about insufficient rights.

Actual behavior
---------------
The error message is empty.

Steps to reproduce
------------------
* create a list with the option "info" set to "conceal"
* open https://yourlistserver.tld/sympa/info/listname in a private browser window
* the red error box pops up, but there's no message in it

**Comment by ikedas** [comment]
Hi @drkspc ,

Can you show the logs _from when_ you opened the info URL _to when_ the error message was shown?

**Comment by drkspc** [comment]
Hi @ikedas,

of course! Here we go:
```
Sep 26 11:32:12 sympa wwsympa[59322]: info main::do_info() [robot lists.example.com] [session 11293295934769] [client 172.16.x.y] [list mylist]
Sep 26 11:32:12 sympa wwsympa[59322]: info main::check_authz() [robot lists.example.com] [session 11293295934769] [client 172.16.x.y] [list mylist] Access denied in do_info for  
Sep 26 11:32:12 sympa wwsympa[59322]: info main::do_home() [robot lists.example.com] [session 11293295934769] [client 172.16.x.y] [list mylist]
```
These are the only log entries concerning that list or my ip within that time frame.

**Comment by ikedas** [comment]
Merged to #1692
